Source/KISAddonConfig.cs

```diff
@@ -22,52 +23,154 @@ sealed class KISAddonConfig : MonoBehaviour {
   const string MaleKerbalEva = "kerbalEVA";
   const string FemaleKerbalEva = "kerbalEVAfemale";
   const string RdKerbalEva = "kerbalEVA_RD";
-  
-  public void Awake() {
-    ConfigAccessor.ReadFieldsInType(GetType(), this);
-    ConfigAccessor.ReadFieldsInType(typeof(ModuleKISInventory), instance: null);
-
-    // Set inventory module for every eva kerbal
-    DebugEx.Info("Set KIS config...");
-    ConfigNode nodeSettings = GameDatabase.Instance.GetConfigNode("KIS/settings/KISConfig");
-    if (nodeSettings == null) {
-      DebugEx.Error("KIS settings.cfg not found or invalid !");
-      return;
+
+  static ConfigNode nodeSettings;
+
+  class KISConfigLoader: LoadingSystem
+  {
+    public KISAddonConfig owner;
+
+    public override bool IsReady ()
+    {
+      return true;
+    }
+
+    public override void StartLoad ()
+    {
+      ConfigAccessor.ReadFieldsInType(owner.GetType(), owner);
+      ConfigAccessor.ReadFieldsInType(typeof(ModuleKISInventory), instance: null);
+
+      // Set inventory module for every eva kerbal
+      DebugEx.Info("Set KIS config...");
+      nodeSettings = GameDatabase.Instance.GetConfigNode("KIS/settings/KISConfig");
+      if (nodeSettings == null) {
+        DebugEx.Error("KIS settings.cfg not found or invalid !");
+      }
     }
+  }
+
+  class KISPodInventoryLoader: LoadingSystem
+  {
+    public bool done;
 
-    // Kerbal parts.
-    UpdateEvaPrefab(MaleKerbalEva, nodeSettings);
-    UpdateEvaPrefab(FemaleKerbalEva, nodeSettings);
-
-    // Set inventory module for every pod with crew capacity.
-    DebugEx.Info("Loading pod inventories...");
-    foreach (AvailablePart avPart in PartLoader.LoadedPartsList) {
-      if (avPart.name == MaleKerbalEva || avPart.name == FemaleKerbalEva
-          || avPart.name == RdKerbalEva
-          || !avPart.partPrefab || avPart.partPrefab.CrewCapacity < 1) {
-        continue;
+    int loadedPartCount;
+    int loadedPartIndex;
+
+    bool HasPodInventories (Part part)
+    {
+      for (int i = part.Modules.Count; i-- > 0; ) {
```

Comment:
This can be written much more compact using `Linkq`. It's used widely in the mod anyways, and the performance in this cases is very similar for the both implementations.
```c#
return part.Modules.ofType<ModuleKISInventory>()
    .Where(m => m.invType == ModuleKISInventory.InventoryType.Pod)
    .Any();
```